contracts/KlerosLiquidProxy.sol

```diff
@@ -0,0 +1,245 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IKlerosLiquidProxy.sol";
+import "./interfaces/IExtendedKlerosArbitrator.sol";
+
+contract KlerosLiquidProxy is IKlerosLiquidProxy {
+    IClaimsManager public immutable override claimsManager;
+
+    IArbitrator public immutable override klerosArbitrator; // Kleros expects this exact name
+
+    bytes public override klerosArbitratorExtraData; // Kleros expects this exact name
+
+    mapping(uint256 => uint256) public override disputeIdToClaimIndex;
+
+    mapping(uint256 => uint256) public override claimIndexToDisputeId;
+
+    modifier onlyClaimForwardedToKleros(uint256 claimIndex) {
+        require(claimIndexToDisputeId[claimIndex] != 0, "Invalid claim index");
+        _;
+    }
+
+    constructor(
+        address _claimsManager,
+        address _klerosArbitrator,
+        bytes memory _klerosArbitratorExtraData,
+        string memory _metaEvidence
+    ) {
+        claimsManager = IClaimsManager(_claimsManager);
+        klerosArbitrator = IArbitrator(_klerosArbitrator);
+        klerosArbitratorExtraData = _klerosArbitratorExtraData;
+        emit MetaEvidence(0, _metaEvidence);
+    }
+
+    function forwardDisputeToKlerosArbitrator(uint256 claimIndex)
+        external
+        payable
+        override
+    {
+        (
+            ,
+            IClaimsManager.ClaimStatus claimStatus,
+            address claimant,
+            ,
+            ,
+            ,
+            string memory evidence
+        ) = claimsManager.claims(claimIndex);
```

Comment:
We're paying to read the entire struct here (even if some fields aren't assigned to variables) so this is more expensive than ClaimsManagerWithKlerosArbitration reading the relevant fields of the struct from the storage

Reply:
[This](https://github.com/api3dao/claims-manager/pull/27#pullrequestreview-1061782163) should resolve the gas cost issue, as the claim arguments won't be read from storage for forwarding

Reply:
To be tracked by https://github.com/api3dao/claims-manager/issues/33